### PR TITLE
Keep validated entries for multi-request and response-profile

### PR DESCRIPTION
**Pull request by tan-tan-kanarek**

### CODE REVIEW

**Comment by tan-tan-kanarek** [comment]
@EladCohenKaltura, please review.

File: alpha/lib/model/entryPeer.php [hunk 1]
```diff
@@ -796,29 +799,46 @@ public static function retrieveByPKs($pks, PropelPDO $con = null)
...
+		self::$validatedEntries[] = $entryId;
+	}
 
-		$dbEntries = self::doSelect($c);
-
-		$entryIds = array();
-		foreach ($dbEntries as $dbEntry)
+	public static function filterEntriesByPartnerOrKalturaNetwork(array $entryIds, $partnerId)
+	{
+		KalturaLog::debug("Valid enties: " . implode(',', self::$validatedEntries));
```
Comment (EladCohenKaltura) [inline alpha/lib/model/entryPeer.php]: Small typo - entries instead of enties

File: alpha/lib/model/entryPeer.php [hunk 2]
```diff
@@ -796,29 +799,46 @@ public static function retrieveByPKs($pks, PropelPDO $con = null)
...
+				$criterionPartnerOrKn = $c->getNewCriterion(entryPeer::PARTNER_ID, $partnerId);
+				$criterionPartnerOrKn->addOr($c->getNewCriterion(entryPeer::DISPLAY_IN_SEARCH, mySearchUtils::DISPLAY_IN_SEARCH_KALTURA_NETWORK));
+				$c->addAnd($criterionPartnerOrKn);
+			}
+	
+			$dbEntries = self::doSelect($c);
+	
+			foreach ($dbEntries as $dbEntry)
+			{
+				$validatedEntries[] = $dbEntry->getId();
```
Comment (EladCohenKaltura) [inline alpha/lib/model/entryPeer.php]: We keep increasing this list - it a member  - wouldn't be good to limit its size ? some sort of sized queue

Comment (tan-tan-kanarek) [inline alpha/lib/model/entryPeer.php]: It's already limited in line 821: $entryIds = array_slice($entryIds, 0, baseObjectFilter::getMaxInValues());

**Pull request merged**